docs/reference/settings/ml-settings.asciidoc

```diff
@@ -81,3 +81,13 @@ sufficient:
 The maximum number of records that are output per bucket. The default value is 
 `500`.
 
+`xpack.ml.max_lazy_ml_nodes`:: (<<cluster-update-settings,Dynamic>>)
+The number of lazily spun up Machine Learning nodes. Useful in situations
+where ML nodes are not desired until the first Machine Learning Job
+is opened. It defaults to `0` and has a maximum acceptable value of `3`.
+If the current number of ML nodes is `>=` than this setting, then it is
+assumed that there are no more lazy nodes available as the desired number
+of nodes have already been provisioned. When a job is opened with this
+setting set at `>0` and there are no nodes that can accept the job, then
+the job will stay in the `OPENING` state until a new ML node is added to the
+cluster and the job is assigned to run on that node.
```

Comment:
Might be worth adding a final couple of sentences saying, "In this scenario, some external process is responsible for adding ML nodes to the cluster. This setting is only useful in conjunction with such an external process."

Reply:
For sure, better to be explicit than to imply